### ISSUE TITLE
Bad download speed

### ISSUE THREAD

**Issue by John-Gee** [zom-ponks/rpcs3-gameupdater#40]
So I just realized that because of resuming, the download speed can look pretty messed up, ex:

Transferred 294/294 Mb (100.00%) at 577728870 Mb/s

(My connection can't go higher than 12Mb/s)

Should we care about that? We can easily fix it, but it'll make the code a bit uglier and I'm not sure if it's necessary.

**Comment by zom-ponks** [comment]
So this is just a glitch in the display (or calculating the numbers) and not actual DL speed problem, right?

**Comment by John-Gee** [comment]
Correct. It's because the bytes used for bytes_downloaded/time are from the current length in bytes, and not just what was downloaded this try.

Saving the initial bytes if the file already exists and using this instead of what grab gives would solve this.
